src/math/lp/.clang-format

```diff
@@ -2,4 +2,4 @@ BasedOnStyle: Google
 IndentWidth: 4
```

Comment:
maybe push this directly as separate update?
There are other changes that are orthogonal to dio that can be pushed.